spec/integration/user_integration_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe 'users tests', type: :system do
+  describe 'users' do
+    subject { User.new(name: 'Tom', email: 'tom@mail.com', password: 'password') }
+    before { subject.save }
+    before(:each) do
+      @category = Group.create(name: 'Category', author_id: subject.id)
+      @dealing = Dealing.create(name: 'transaction', amount: 15, author_id: subject.id)
+      visit root_path
+    end
+
+    it 'shows app name' do
+      expect(page).to have_content('ExpenseWatch')
+    end
+
+    it 'shows login button' do
+      expect(page).to have_content('Login')
+    end
+
+    it 'shows signup button' do
+      expect(page).to have_content('Sign up')
+    end
+
```

Comment:
There's an error in the `user_integration_spec.rb` file `line: 18` and `line:22`. 

![image](https://user-images.githubusercontent.com/38088818/229106815-97aba07b-26b7-4103-9b15-4548c8f88aec.png)

- This is because the test expects buttons with the text `Login` and `Sign up`, yet the buttons on the splash screen have the text `LOGIN` and `SIGN UP`.

![image](https://user-images.githubusercontent.com/38088818/229107492-b3cfc585-740b-4794-a500-02562c86dd5e.png)

Note: The Capybara search is `case sensitive`, therefore when you use `expect(page).to have_content('...')` the text should be an exact match as it is on the page including cases.